### PR TITLE
Allow netcoreapp1.1 as a valid TestTFM

### PR DESCRIPTION
**Pull request by ianhays**
Updates buildtools and BuildTest versions to consume some changes that allow netcoreapp1.1 as a TestTFM. <strike>I also modified the CoreFX targets to use netcoreapp1.1 as the new default.</strike>

to be merged after https://github.com/dotnet/corefx/pull/10423/files

@ericstj @weshaggard @karajas

### CODE REVIEW

File: dir.props [hunk 1]
```diff
@@ -520,9 +527,9 @@
 
   <!-- Default Test platform to deploy the netstandard compiled tests to -->
   <PropertyGroup>
-    <TestTFM Condition="'$(TestTFM)'==''">netcoreapp1.0</TestTFM>
-    <!-- we default FilterToTestTFM to netcoreapp1.0 if it is not explicity defined -->
-    <FilterToTestTFM Condition="'$(FilterToTestTFM)'==''">netcoreapp1.0</FilterToTestTFM>        
+    <TestTFM Condition="'$(TestTFM)'==''">netcoreapp1.1</TestTFM>
```
Comment (weshaggard) [inline dir.props]: Do these tests even work for netcoreapp1.1 yet? Right now I'm not sure I want to move our default to netcoreapp1.1 for the entire repo yet because it isn't clear if that is the TFM we will use for our 1.1 release (despite the numbers matching :()

File: dir.props [hunk 2]
```diff
@@ -452,6 +453,12 @@
         <NuGetTargetMoniker>.NETCoreApp,Version=v1.0</NuGetTargetMoniker>
       </PropertyGroup>
     </When>
+    <When Condition="'$(TargetGroup)'=='netcoreapp1.1'">
```
Comment (joperezr) [inline dir.props]: ><When Condition="'$(TargetGroup)'=='netcoreapp1.1'"> [](start = 4, length = 52)

Given netstandard1.7 resolves to netcoreapp1.1, should we also add a `<When>` block for netstandard1.7 here?

Comment (ianhays) [inline dir.props]: we could (and eventually should), but probably in a different PR as there are some other places we would need to add netstandard1.7 entries as well. afaik we're not using netstandard1.7 anywhere in CoreFX yet.

Comment (joperezr) [inline dir.props]: we are in dev/api branch which is why I asked this :smile: , but that's ok, we can do it in a different PR.

Comment (ianhays) [inline dir.props]: In that case, yeah we should. And another thing we'd want to do is modify some of the targets in buildtools to assign netcoreapp1.1 as the default TestTFM when under netstandard1.7. Right now there isn't really a relationship between the two.

**Comment by ianhays** [comment]
@weshaggard I left the default as netcoreapp1.0 but added an easy way to set the TestTFM to netcoreapp1.1.

**Comment by ianhays** [comment]
@dotnet-bot test  Innerloop CentOS7.1 Debug Build and Test please

File: dir.props (hunk 2 above)
Comment (joperezr) [inline dir.props]: I did this in dev/api branch already here: https://github.com/dotnet/corefx/blob/dev/api/dir.props#L524

**Comment by weshaggard** [comment]
LGTM

**Comment by ianhays** [comment]
Thanks for taking a look!
